algebraic-graphs.cabal

```diff
@@ -147,7 +147,7 @@ test-suite test-alga
     if !impl(ghc >= 8.0)
         build-depends:  semigroups   >= 0.18.2  && < 0.18.4
     if impl(ghc >= 8.0.2)
-        build-depends:  inspection-testing >= 0.4 && < 0.5
+        build-depends:  inspection-testing >= 0.4 && < 0.4.2
```

Comment:
Has the rebase gone wrong? This shouldn't be here.

Reply:
Probably, I didn't change it (and it's already like this on master)

Reply:
I think I corrected it